catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/app/UserApplication.java

```diff
@@ -52,6 +61,12 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(UserApplication.class);
 
+  private static final String MAX_PAGE_SIZE_PROPERTY = "catalog.maxPageSize";
+
+  private static final Integer DEFAULT_MAX_PAGE_SIZE = 1000;
+
+  public static final Integer MAX_PAGE_SIZE = determineAndRetrieveMaxPageSize();
```

Comment:
✏️ Instead of using a static variable we should just call `determineAndRetrieveMaxPageSize()` in `getSubjectNotifications()` so it behaves properly when the value of the system property changes, which could happen at any time.